test_ai_aggies.py

```diff
@@ -12,7 +11,33 @@ def test_show_aggie_pride(self):
     def test_anonymize_nrp(self):
         """Test to make sure NRP is anonymized"""
         self.assertEqual('My political affiliation is <NRP>',
-                         anonymize_text('My political affiliation is Republican', ['NRP']))
+                         anonymize_text('My political affiliation is Democrat', ['NRP']))
 
         self.assertEqual('My NRP is Unknown',
                          anonymize_text('My NRP is Unknown', ['NRP']))
+
+    def test_person(self):
+        """Test to ensure a Person is recognized and anonymized correctly"""
+        # Positive test case - Person should be anonymized
+        test_person = 'Nicholas Dewitt'
+        test_string = 'My name is ' + test_person
+        expected_result = 'My name is <PERSON>'
+        actual_result = anonymize_text(test_string, ['PERSON'])
+        self.assertEqual(expected_result,
+                        actual_result)
+
+        # Negative test case - Person should not be anonymized if not in the list
```

Comment:
Your comment says this is a negative test case, however, it looks like this is actually a positive test case.  The comment should be updated to reflect this.

Reply:
Yeah the incorrect negative test case seems to be the only thing that needs correction, I will approve the merge after it is fixed.